Sources/MapboxNavigation/CarPlayManager.swift

```diff
@@ -451,29 +469,46 @@ extension CarPlayManager {
         previewRoutes(between: [origin, destination], completionHandler: completionHandler)
     }
     
+    /**
+     Allows to preview routes for a list of `Waypoint` objects.
+     
+     - parameter waypoints: A list of `Waypoint` objects.
+     - parameter completionHandler: A closure to be executed when the calculation completes.
+     */
     public func previewRoutes(between waypoints: [Waypoint], completionHandler: @escaping CompletionHandler) {
         let options = NavigationRouteOptions(waypoints: waypoints)
         previewRoutes(for: options, completionHandler: completionHandler)
     }
     
+    /**
+     Allows to preview routes based on criteria, which is specified in `RouteOptions` object.
+     
+     - parameter routeOptions: A `RouteOptions` object, which specifies the criteria for results
+     returned by the Mapbox Directions API.
+     - parameter completionHandler: A closure to be executed when the calculation completes.
```

Comment:
Would it be more accurate to say that this closure gets executed once the preview is fully visible?

Reply:
Yep, agree.